content/en/docs/reference/networking/virtual-ips.md

```diff
@@ -262,7 +262,7 @@ exits with an error.
 
 ### `nftables` proxy mode {#proxy-mode-nftables}
 
-{{< feature-state for_k8s_version="v1.29" state="alpha" >}}
+{{< feature-state for_k8s_version="v1.31" state="beta" >}}
```

Comment:
Improvement for 'feature-state' shortcode usage here, consider using the new `feature_gate_name` option, making the feature state more data-driven.

```suggestion
{{< feature-state feature_gate_name="NFTablesProxyMode" >}}
```

Reply:
nice!